.travis.yml

```diff
@@ -14,16 +14,17 @@ env:
   global:  # default values
     - PYTHON_VERSION=3.7
   matrix:
-    - TASK=regular PYTHON_VERSION=3.6
-    - TASK=sdist PYTHON_VERSION=2.7
-    - TASK=bdist
-    - TASK=if-else
-    - TASK=lint
-    - TASK=check-docs
-    - TASK=mpi METHOD=source
-    - TASK=mpi METHOD=pip
-    - TASK=gpu METHOD=source PYTHON_VERSION=3.5
-    - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
+    # - TASK=regular PYTHON_VERSION=3.6
+    # - TASK=sdist PYTHON_VERSION=2.7
+    # - TASK=bdist
+    # - TASK=if-else
+    # - TASK=lint
+    # - TASK=check-docs
+    # - TASK=mpi METHOD=source
+    # - TASK=mpi METHOD=pip
+    # - TASK=gpu METHOD=source PYTHON_VERSION=3.5
+    # - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
+    - TASK=r-pkg R_VERSION=3.6.1
```

Comment:
right now we'll get this R version when building on Mac, but on Ubuntu we're at the whim of whatever is "latest" for the particular ubuntu version we are running. In this case the latest version for R available on bionic is 3.4.4 (see [here](https://packages.ubuntu.com/search?keywords=r-base)). You can see [here](https://cran.r-project.org/bin/linux/ubuntu/#installation) for more.

I think that ok is ok for now.

Reply:
Then this variable should be named something like `R_VERSION_MAC` to avoid any misleading interpretation.

BTW, why did you abandoned the idea to install the latest version from CRAN sources for Ubuntu?

Do we have any limitation in using chars here? 😄 `TASK=r-package` seems to be more pretty to see.

Reply:
ha nope no limitation, happy to use `r-package`. `r-pkg` is a bad habit of mine from other projects.

> why did you abandoned the idea to install the latest version from CRAN sources for Ubuntu

I followed the instructions from https://cran.r-project.org/bin/linux/ubuntu/README.html which SAY you should get 3.6.x, but still ended up with 3.4.x. I'm not sure why. We are using bionic (Ubuntu 18.04) on Travis, and it seems [3.4.x is the latest `r-base` release available](https://packages.ubuntu.com/search?keywords=r-base). I think I see the problem though...found [in this answer](https://stackoverflow.com/a/56378217/3986677) that there is one more step to see more versions from that server. I think in my latest commit, I have it successfully installing 3.6.1